libraries/app/database_api.cpp

```diff
@@ -198,6 +198,24 @@ class database_api_impl : public std::enable_shared_from_this<database_api_impl>
          return tmp;
       }
 
+      const account_object* get_account_from_string( const std::string& name_or_id  )
```

Comment:
I do not believe this method changes anything internally. So you should be able to mark this const, and keep the const qualifier on the methods that call it.

Reply:
hey John, thank you for that. i had the intention of fix that before submitting but it seems i forgot. fixed now at https://github.com/bitshares/bitshares-core/pull/989/commits/30eda4afc2bb1d4dec3ba9d634b3198c90763cab thanks again!